stack-nix-ghc9.yaml

```diff
@@ -0,0 +1,58 @@
+resolver: nightly-2022-06-10
+system-ghc: true
+install-ghc: false
+nix:
+  enable: false
+packages:
+  - kore
+extra-deps:
+  - ghc-trace-events-0.1.2.1
+  - monoidal-containers-0.6.0.1
+  - newtype-0.2.2.0
+  - sqlite-simple-0.4.18.0
+  - direct-sqlite-2.3.26
+  - witherable-0.4.2
+  - witherable-class-0
+  - git: https://github.com/co-log/co-log.git
+    commit: 9227c7f2b06813c8a1354b1057255fab3c885c6b
+    # nix-sha256: 0ahj3w2zkn31wjqgaax0s0g6y0qc44ysc2ysqkz24s4l4gkkhg6q
+    subdirs:
+      - co-log-core
+      - co-log
+  - git: https://github.com/andrewthad/chronos.git
+    commit: 13b46574f2d811f27c693c78d92aed71c82f39d5
+  - git: https://github.com/andrewthad/contiguous.git
+    commit: 28aaa3ee216ff58c712a3393435a0da62e057cca
+  - git: https://github.com/andrewthad/run-st.git
+    commit: 74c8e13d59d936559b22798415c011a5a0d2570b
+  - git: https://github.com/kowainik/typerep-map.git
+    commit: 41838d2c1b3626bc2e7ae560bf85d3280b1cda20
+    # nix-sha256: 1w5vwf12z3blpnq5sw9br2bi8w81hmzxizcz69x1z6q38cqygzrz
+  - git: https://github.com/byteverse/bytebuild.git
+    commit: 59bb9596086213c70f98c7599a14be52685fd321
+  - git: https://github.com/tek/bytesmith.git
+    commit: 9237825bf757cc9d46365a341bb51004b68f84a0
+    # nix-sha256: 1rscv4p071ff7lk3jl3xfjzdjk08qnjv5rk5vf3rg0g7fd0j4gcb
+  - git: https://github.com/byteverse/byteslice.git
+    commit: 5c1ba9fada01b5101e6aeaeb0d5d37aff852b1ba
+  - git: https://github.com/byteverse/zigzag.git
+    commit: dcf660fd76f134fccd550ce4c30bb26d4fb31b42
+  - ghc-events-0.17.0.3
+  - tasty-test-reporter-0.1.1.4
+  - junit-xml-0.1.0.0
+  - compact-0.2.0.0
+  - git: https://github.com/goodlyrottenapple/pipes-ghc-events.git
+    commit: 0b6b43a5d6547cc3ca394d095bd551befb4d0b4e
+    subdirs:
+      - eventlog2speedscope
+      - pipes-aeson
+      - pipes-ghc-events
+    # nix-sha256: 17bw2nbyz3nxc9ijcs5bkn4p0dh25jnvid7ps81zyh52l0551kjb
```

Comment:
This sparked my curiosity... assume it is a fork of ttuegel/pipes-ghc-events ?
I am unable to access the repo, which might be a problem...

Reply:
thanks for the spot! i deleted it thinking i don't need it but had it cached. recreated now